docs/fetching_incidents/README.md

```diff
@@ -60,11 +60,19 @@ Incidents are created by building an array of incident objects. These object all
 
 ```python
     # convert the events to demisto incident 
+    events = [
+        {
+            'name': 'event_1',
+            'create_time': '2019-10-23T10:11:00Z',
+            'event_id': 100
+        }
+    ]
+    
     incidents = []
     for event in events:
         incident = {
             'name': event['name'],        # name is required field, must be set
-            'occurred': event['create_time'], # occurred is optional date
+            'occurred': event['create_time'], # occurred is optional date - must be string of a format ISO8601
```

Comment:
i think we should mention the date format in the doc, and not just as a comment in the code